Resources/public/style/httplug.css

```diff
@@ -80,6 +80,11 @@
     font-size: 12px;
 }
 
+.httplug-duration {
+    min-width: 6ch;
```

Comment:
I use `number_format` twig filter, so you get a thousand separator. But anyway, I set this as a `min-width` to get the same width when duration is lower than 100ms and lower than 1000ms. When duration is over 1000ms, the box get larger.